mtp_noms_ops/assets-src/stylesheets/views/_checks.scss

```diff
@@ -73,3 +73,84 @@ table.mtp-table--review {
 .button-warning {
   @include button($red);
 }
+
+// TODO remove once new gov.uk styling implemented
+.moj-sub-navigation {
+  margin-bottom: 0.7rem;
+}
+
+
+.moj-sub-navigation__list {
+  font-size: 0; // Removes white space when using inline-block on child element.
+  box-shadow: inset 0 -1px 0 #b1b4b6;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+}
+
+
+.moj-sub-navigation__item {
+  line-height: 1.31579;
+  display: inline-block;
+
+  font-family: "Helvetica Neue", "Inter UI", sans-serif;
```

Comment:
i'd remove this line to inherit the normal font

Reply:
As long as we're happy with this differing from Justin's designs, then sure

Reply:
the font used on gov.uk isn't licensed for use in some places and that prototype does not include it